### PR TITLE
copied all new files

### PR DESCRIPTION
**Pull request by shuhaib-aot**
Changes between v2.3.0 Formio version Vs formsflow.ai / formsflow-forms version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,12 @@
 # Use Node image, maintained by Docker:
 # hub.docker.com/r/_/node/
 FROM node:lts-alpine3.10
-WORKDIR /app
+
+# set working directory
+WORKDIR /forms-flow-forms/app
```
Comment (sumesh-aot) [inline Dockerfile]: this have to change as there is no forms-flow-forms folder in fork

File: Dockerfile [hunk 2]
```diff
@@ -5,7 +5,12 @@
...
+
+RUN set -x \
+	&& chmod -R 777 /forms-flow-forms/app/
```
Comment (sumesh-aot) [inline Dockerfile]: here too

File: Dockerfile [hunk 3]
```diff
@@ -19,34 +24,43 @@ RUN apk update && \
...
-ENV NPM_PACKAGES=/.npm-packages \
-    PATH=$NPM_PACKAGES/bin:$PATH \
-    NODE_PATH=$NPM_PACKAGES/lib/node_modules:$NODE_PATH
-RUN echo "prefix = $NPM_PACKAGES" >> ~/.npmrc
-
-# Include details of the required dependencies
-COPY ./package.json $NPM_PACKAGES/
-COPY ./package-lock.json $NPM_PACKAGES/
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /forms-flow-forms/app/node_modules/.bin:$PATH
```
Comment (sumesh-aot) [inline Dockerfile]: same here

File: docker-compose-linux.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Allows us to start the stack without touching too much of our local system
```
Comment (sumesh-aot) [inline docker-compose-linux.yml]: docker-compose can stay in forms-flow-ai/forms-flow-forms folder

File: openshift_custom_Dockerfile [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+# if the application is built from a source repo other than aot , use the custom copying.
+# steps : checkout the aot source code and use the copy logic.
+# -----------------------------------------------------------
+
+# Use Node image, maintained by Docker:
+# hub.docker.com/r/_/node/
+FROM docker.io/node:12.2.0-alpine
+
+# set working directory
+WORKDIR /forms-flow-forms/app
```
Comment (sumesh-aot) [inline openshift_custom_Dockerfile]: changes needed here too

File: script/resourceId_linux.sh [hunk 6]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```
Comment (sumesh-aot) [inline script/resourceId_linux.sh]: these script too we can keep in forms-flow-ai repo

File: Dockerfile (hunk 1 above)
Comment (abhilash-aot) [reply Dockerfile]: This folder is created inside Docker container image  right ? Should work with forms-flow-forms So could use Same. But changing to /app should also work

File: docker-compose-linux.yml (hunk 4 above)
Comment (abhilash-aot) [reply docker-compose-linux.yml]: ok Wil Update , We can have a single docker-compose in this formio Repo so we can build the forms in here as well . We can have a CI/ CD process to get the image pushed to dockerhub and that image will be used by the forms-flow-ai Git Repo

File: script/resourceId_linux.sh (hunk 6 above)
Comment (abhilash-aot) [reply script/resourceId_linux.sh]: Yup that makes more sense. As the role ID context is suited in forms-flow-ai

**Review by abilpraju-aot — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1,130 +1,175 @@
...
+* The value of Mongo database details (especially if this instance is not just for testing purposes)
+* The value of ROOT user account details (especially if this instance is not just for testing purposes)
+  
+### Running the application
+
+* forms-flow-forms service uses port 3001, make sure the port is available.
+* `cd {Your Directory}/forms-flow-ai/forms-flow-forms`
+
+* For Linux,
+  * Run `docker-compose -f docker-compose-linux.yml up -d` to start.
```
Comment (abhilash-aot) [inline README.md:66]: Update readme. We can have a single docker-compose file

**Review by abhilash-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**